rubyforgood/diaper
# 1329 add donation money raised column

Resolves #1329

### Description
This PR addresses issue #1329 . I added a money raised column to the donations page and a total at the bottom. I removed the variety of items column.  I followed patterns I found in the project for calculating the total and testing it. The money raised form field on the new/edit page is a plain integer field, so the user has to enter in the amount in pennies. That's not intuitive. It should probably accept decimals, or auto-format to a currency value. However, changing that would be outside the scope of this issue. 

### Type of change

* New feature (non-breaking change which adds functionality)
* Breaking change (fix or feature that would cause existing functionality to not work as expected)

### How Has This Been Tested?
None of the seeded donations have a money raised value. 
1. Go to the donations page and confirm that the page displays and no values are shown.
2. Add a money raised value to one of the donations. Confirm that it shows on the donations index page and the value is at the bottom.
3. Add another money raised value to a different donations. Confirm that both show on the index page and that the total is correctly calculated.
